dotCMS/src/integration-test/java/com/dotcms/graphql/DotGraphQLHttpServletTest.java

```diff
@@ -0,0 +1,34 @@
+package com.dotcms.graphql;
+
+import static org.junit.Assert.*;
+import java.util.HashMap;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import com.dotcms.util.IntegrationTestInitService;
+
+public class DotGraphQLHttpServletTest {
+
+    
+    @BeforeClass
+    public static void prepare() throws Exception{
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5609581)